sdk/src/main/java/com/qonversion/android/sdk/internal/user/generator/UserIdGenerator.kt

```diff
@@ -0,0 +1,6 @@
+package com.qonversion.android.sdk.internal.user.generator
+
+interface UserIdGenerator {
+
+    fun generateUserId(): String
```

Comment:
Do we expect here other functions? If don't, maybe we we can rename function as 'generate'?